bqschema_suite_test.go

```diff
@@ -1,10 +1,10 @@
-package bqschema_test
+package bqschema
```

Comment:
I don't feel strongly that the tests must be in `package bqschema` rather than `bqschema_test` so I'll change this to use the dot import style.